tests/integration/query.ts

```diff
@@ -118,3 +118,83 @@ describe('ensure correct results from latest request', () => {
     expect(updateResult.mock.calls).toHaveLength(2);
   });
 });
+
+
+let uuidString = 'some-uuid-value';
+const mockUuid = jest.fn(() => uuidString);
+jest.mock('uuid', () => ({
```

Comment:
would it be possible to do this with a `__mocks__/uuid` file, kind of like `__mocks__/@yext/answers-core`?

Reply:
update: no longer required. it's removed